TheMovieDb/HomeViewController.swift

```diff
@@ -7,27 +7,60 @@
 
 import UIKit
 
-class HomeViewController: UIViewController, UICollectionViewDelegate {
+final class HomeViewController: UICollectionViewController {
     var viewMoreButton = UIButton()
-    var collectionView: UICollectionView!
+
     let service = NetworkManager(urlSession: URLSession.shared)
```

Comment:
make it private since it's a dependency and inject over init to allow dependency injection